db/migrate/20180118002015_create_datasets_v2.rb

```diff
@@ -0,0 +1,39 @@
+class CreateDatasetsV2 < ActiveRecord::Migration
+  def change
+    create_table :datasets do |t|
+
+      t.string :name
+      t.text :description
+      t.integer :creator_id
+      t.timestamps null: false
+
+    end
+    create_table :dataset_items do |t|
+
+      t.integer :dataset_id
+      t.integer :audio_recording_id
+      t.decimal :start_time_seconds, :null => false
+      t.decimal :end_time_seconds, :null => false
+      t.decimal :order
+      t.datetime :created_at
```

Comment:
Tables that have :created_at should also have :creator_id

Reply:
I don't think we were interested in adding a :creator_id here. @peichins do you remember why?